cookbook/deployment/workflow/lp_notifications.py

```diff
@@ -1,16 +1,124 @@
+# 
+# In order to process notifications, you'll need to set up an `AWS SQS <https://aws.amazon.com/sqs/>`_ queue to consume notification events. This queue must be configured as a subscription to your SNS topic you created above.
+#
+# In order to actually publish notifications, you'll need a `verified SES email address <https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-addresses-and-domains.html>`_ which will be used to send notification emails and alerts using email APIs.
+#
+# The role you use to run Flyteadmin must have permissions to read and write to your SNS topic and SQS queue.
+#
+# Let's look into the following config section and explain what each value represents: ::
+#
+  notifications:
+    type: "aws"
+    region: "us-east-1"
+    publisher:
+      topicName: "arn:aws:sns:us-east-1:{{ YOUR ACCOUNT ID }}:{{ YOUR TOPIC }}"
+    processor:
+      queueName: "{{ YOUR QUEUE NAME }}"
+      accountId: "{{ YOUR ACCOUNT ID }}"
+    emailer:
+      subject: "Notice: Execution \"{{ workflow.name }}\" has {{ phase }} in \"{{ domain }}\"."
+      sender:  "flyte-notifications@company.com"
+      body: >
+        Execution \"{{ workflow.name }} [{{ name }}]\" has {{ phase }} in \"{{ domain }}\". View details at
+        <a href=\http://flyte.company.com/console/projects/{{ project }}/domains/{{ domain }}/executions/{{ name }}>
+        http://flyte.company.com/console/projects/{{ project }}/domains/{{ domain }}/executions/{{ name }}</a>. {{ error }}
+
+# * **type**: because AWS is the only cloud back-end supported for executing scheduled workflows, only ``"aws"`` is a valid value. By default, the no-op executor is used.
+# * **region**: this specifies which region AWS clients should use when creating SNS and SQS clients.
+# * **publisher**: This handles pushing notification events to your SNS topic.
+#     * **topicName**: This is the arn of your SNS topic
+# * **processor**: This handles the recording notification events and enqueueing them to be processed asynchronously.
+#     * **queueName**: This is the name of the SQS queue which will capture pending notification events
+#     * **accountId**: Your AWS `account id <https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#FindingYourAWSId>`_
+# * **emailer**: This section encloses config details for sending and formatting emails used as notifications.
+#     * **subject**: Configurable subject line used in notification emails
+#     * **sender**: Your verified SES email sender
+#     * **body**: Configurable email body used in notifications
+#
+# The full set of parameters which can be used for email templating are checked into `code <https://github.com/flyteorg/flyteadmin/blob/a84223dab00dfa52d8ba1ed2d057e77b6c6ab6a7/pkg/async/notifications/email.go#L18,L30>`_.
+#
+.. _admin-config-example:
+
+Example config
```

Comment:
Please check if this example is suitable here, since there are 2 more examples below.

Reply:
Updated the file.